src/nbb/impl/repl.cljs

```diff
@@ -170,6 +170,8 @@
   ([] (repl nil))
   ([_opts]
    (when tty (.setRawMode js/process.stdin true))
+   (doseq [form nbb/repl-requires]
```

Comment:
This should be implement async, because eval-require itself is async, similar to above.

Reply:
you mean sync like with the .then chain right

Reply:
e8ced5773407813f922af5eb0349e9412ecd33a0